core/migrations/0004_copyright_lccndatecopyright.py

```diff
@@ -26,7 +26,7 @@ class Migration(migrations.Migration):
                 ('lccn', models.CharField(max_length=25)),
                 ('start_date', models.DateField()),
                 ('end_date', models.DateField()),
-                ('copyright', models.ForeignKey(related_name='lccn_date_copyright', to='core.Copyright')),
+                ('copyright', models.ForeignKey(related_name='lccn_date_copyright', to='core.Copyright', on_delete=models.CASCADE)),
```

Comment:
is this the best place to do this in django?  At least from my time with Ruby on Rails, I would expect that rather than editing an existing migration, that one would create a new migration that changed this field.  How would this affect apps which are already running with a database?

Reply:
From what I read, we had to add it in both places because newly spinning it up (for dev or new adopters) would fail without that.  Django 2 no longer allows the implicitly defined `on_delete` value, so migrations simply refuse to run without that value.

So short answer would be "yes" :P

Reply:
great reason